core/arch/arm/plat-amlogic/platform_config.h

```diff
@@ -0,0 +1,26 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2020 Carlo Caione <ccaione@baylibre.com>
+ */
+
+#ifndef PLATFORM_CONFIG_H
+#define PLATFORM_CONFIG_H
+
+#include <mm/generic_ram_layout.h>
+
+/* Make stacks aligned to data cache line length */
+#define STACK_ALIGNMENT		64
+
+#ifdef ARM64
+#ifdef CFG_WITH_PAGER
+#error "Pager not supported for ARM64"
+#endif
+#endif /*ARM64*/
```

Comment:
Is this true? Generally speaking OP-TEE supports `CFG_WITH_PAGER=y` on Arm64, I suspect those lines were copied from older code.

If you don't want pager enabled (e.g., you know it doesn't work with your platform or for whatever reason!), I'd rather have `$(call force,CFG_WITH_PAGER,n)` in `plat-amlogic/conf.mk`.

Reply:
Yeah, it was copied over and at the moment the platform doesn't use the pager so I'll set that to `n`. Thanks.